src/api/status/src/js/feed-stats.js

```diff
@@ -0,0 +1,16 @@
+const { fetch } = require('@senecacdot/satellite');
+
+const apiUrl = `https://${process.env.API_HOST}`;
```

Comment:
This will break on localhost.  Why not just use `API_URL`?

Reply:
Just wondering do I have access to the global .env, because when I call `process.env.API_URL` it is undefined.
Do I declare a new environment  variable for `API_URL` in `src\api\status\env.local`

Reply:
You have access to whatever environment variables get loaded into the process.  So it depends how you run it.  In Docker we use the configs in `config/` and if you are running with `npm start` or something, we use `env.local` as a way to load a similar set of environment variables.  If something is missing from it for this use case, add it.